include/oboe/Stream.h

```diff
@@ -14,31 +14,31 @@
  * limitations under the License.
  */
 
-#ifndef OBOE_OBOE_STREAM_H_
-#define OBOE_OBOE_STREAM_H_
+#ifndef OBOE_STREAM_H_
+#define OBOE_STREAM_H_
 
-#include <stdint.h>
-#include <time.h>
-#include "oboe/OboeDefinitions.h"
-#include "oboe/OboeStreamCallback.h"
-#include "oboe/OboeStreamBase.h"
+#include <cstdint>
+#include <ctime>
+#include "oboe/Definitions.h"
+#include "oboe/StreamBuilder.h"
+#include "oboe/StreamBase.h"
 
 /** WARNING - UNDER CONSTRUCTION - THIS API WILL CHANGE. */
 
-class OboeStreamBuilder;
+namespace oboe {
 
-#define DEFAULT_TIMEOUT_NANOS    (2000 * OBOE_NANOS_PER_MILLISECOND)
+constexpr int64_t kDefaultTimeoutNanos = (2000 * kNanosPerMillisecond);
 
 /**
  * Base class for Oboe C++ audio stream.
  */
-class OboeStream : public OboeStreamBase {
+class Stream : public StreamBase {
```

Comment:
We might want to consider calling this `AudioStream` since `Stream` is quite a generic term. At some point in the future we might also have a `MidiStream`